resources/function_help/json/convert_timezone

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "convert_timezone",
+  "type": "function",
+  "groups": ["Conversions", "Date and Time"],
+  "description": "Converts a datetime object to a different timezone.",
+  "arguments": [{
+    "arg": "datetime",
+    "description": "datetime value"
+  },
+  {
+    "arg": "timezone",
+    "description": "target timezone"
+  }],
+  "examples": [{
+    "expression": "convert_timezone(\"DATE_FIELD\", timezone_from_id('Australia/Darwin'))",
```

Comment:
@nyalldawson Would it be possible to have examples with actual datetime? I don't know if it would be straightforward to write/read but it could help users figure out the expected formatting for the input data. and examples with real data are often more meaningful.